applications/nrf_desktop/doc/hid_provider_keyboard.rst

```diff
@@ -7,10 +7,9 @@ HID provider keyboard module
    :local:
    :depth: 2
 
-The HID provider keyboard module is responsible for providing keyboard HID reports.
-The module listens to the :c:struct:`button_event` event and communicates with the :ref:`nrf_desktop_hid_state`.
-It provides new keyboard HID reports when requested by the :ref:`nrf_desktop_hid_state`.
-It also notifies the :ref:`nrf_desktop_hid_state` when new data is available.
```

Comment:
I don't like the essence of the module being overshadowed by some generic not very informative sentences.

Reply:
I tried to guide reader to the HID state doc section that describes HID providers integration in a generic way. I think it could allow to get better understanding of how HID providers work. It also removes duplications nicely.

Reply:
Ok, rephrased.